src/Xamarin.Android.Build.Tasks/Utilities/ELFHelper.cs

```diff
@@ -66,7 +72,7 @@ static void AssertValidLibraryAlignment (TaskLoggingHelper log, uint pageSize, s
 				log.LogDebugMessage ($"    expected segment alignment of 0x{pageSize:x}, found 0x{segment64.Alignment:x}");
 
 				(string packageId, string packageVersion) = GetNugetPackageInfo ();
-				log.LogCodedWarning ("XA0141", Properties.Resources.XA0141, packageId, packageVersion, Path.GetFileName (path));
+				log.LogCodedWarning ("XA0141", Properties.Resources.XA0141, packageId, packageVersion, Path.GetFileName (path), archName);
```

Comment:
Could we use the overload for `LogCodedWarning` where you pass the file path in, and put 0 for line number:

* https://github.com/dotnet/android-tools/blob/60fae1924e2d71f31dc1ed05f7346fd7874d6636/src/Microsoft.Android.Build.BaseTasks/MSBuildExtensions.cs#L132

This would include the file, and if you double-clicked the error in an IDE it would open the file.